File: api/Params/Admin/AudioMessageParams.cs
namespace api.Params.Admin
{
    public class AudioMessageParams: PaginationParams
    {
        public string RecipientUsername { get; set; }
        public string SenderUsername { get; set; }
        public string CandidateName { get; set; }
        public int ApplicationNo { get; set; }
        public DateTime DateComposed { get; set; }
        public int FeedbackReceived { get; set; }
    }
}

File: client/src/app/_models/params/audioMessageParams.ts
export interface IAudioMessageParams
{
    recipientUsername: string;
    senderUsername: string;
    candidateName: string;
    applicationNo: number;
    dateComposed: Date;
    feedbackReceived: number;
    pageNumber: number;
    pageSize: number;
}

export class audioMessageParams implements IAudioMessageParams
{
    recipientUsername = '';
    senderUsername = '';
    candidateName = '';
    applicationNo = 0;
    dateComposed= new Date();
    feedbackReceived = 0;
    pageNumber=1;
    pageSize=15;
}
